x-pack/qa/security-migrate-tests/src/test/java/org/elasticsearch/xpack/security/MigrateToolTestCase.java

```diff
@@ -68,14 +71,15 @@
     private static Client startClient(Path tempDir, TransportAddress... transportAddresses) {
         logger.info("--> Starting Elasticsearch Java TransportClient {}, {}", transportAddresses, tempDir);
 
-        Settings clientSettings = Settings.builder()
+        Settings.Builder clientSettingsBuilder = Settings.builder()
                 .put("cluster.name", "qa_migrate_tests_" + counter.getAndIncrement())
                 .put("client.transport.ignore_cluster_name", true)
                 .put("path.home", tempDir)
-                .put(SecurityField.USER_SETTING.getKey(), "transport_user:x-pack-test-password")
-                .build();
-
-        TransportClient client = new PreBuiltXPackTransportClient(clientSettings).addTransportAddresses(transportAddresses);
+                .put(SecurityField.USER_SETTING.getKey(), "transport_user:x-pack-test-password");
+        if (Boolean.parseBoolean(System.getProperty(FIPS_SYSPROP)) && JavaVersion.current().getVersion().get(0) == 8) {
```

Comment:
Can't you just do this? I tried the import against your branch and it was OK.

```suggestion
        if (inFipsSunJsseJvm()) {
```

Reply:
https://github.com/elastic/elasticsearch/pull/52211#issuecomment-585092157